### DISCUSSION TITLE
Video Quality

### DISCUSSION THREAD

**Discussion by LadnerJamie** [jmbannon/ytdl-sub#630] [Q&A] [answered]
First off, I just wanted to say thank-you for creating this, I am loving being able to download my favorite YT channels and watch them in Jellyfin and my Kodi install.

I am having some playback issues with 4K videos on my older TV and am wondering if there is a way to specify ytdl-sub to grab 1080 instead of 4K? I see that ytdl-core has the -f option, but I'm not sure how to call it in my config.yaml with ytdl-sub. Is this possible?

**Comment by jmbannon** [comment, marked as answer]
`ytdl_options.format` can take in any yt-dlp format field, where you can limit it to 1080p

**Comment by jmbannon** [comment]
And you're welcome! Glad to see others enjoying YouTube videos without YouTube :smile:

**Comment by catduckgnaf** [comment]
Check out the latest release, with new format and preset for max 1080p!

Add the following preset to download the best available audio and video quality, with the video not greater than 1080p, and remux it into an MP4 container:

* ``max_1080p``

**Reply by LadnerJamie** [reply]
Thanks for the heads-up, I'm going to give this a shot. I have been using 
```
ytdl_options:
      format: "248+bestaudio"
```
But I'll switch over to max_1080p and see how that works.

**Reply by catduckgnaf** [reply]
Thanks! Please let us know how it works
